### DISCUSSION TITLE
Adguard deploy error

### DISCUSSION THREAD

**Discussion by Calzor-Suzay** [tteck/Proxmox#3487] [Help Needed] [answered]
Trying to deploy Adguard to my Proxmox install 

Get this error, can anyone assist?
`    ___       __                           __
   /   | ____/ /___ ___  ______ __________/ /
  / /| |/ __  / __  / / / / __  / ___/ __  / 
 / ___ / /_/ / /_/ / /_/ / /_/ / /  / /_/ /  
/_/  |_\__,_/\__, /\__,_/\__,_/_/   \__,_/   
            /____/                           
 
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 101
Using Hostname: adguard
Using Disk Size: 2GB
Allocated Cores 1
Allocated Ram 512
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Adguard LXC using the above default settings
 ✓ Using qNap for Template Storage.
 ✓ Using vm_data for Container Storage.
 ✓ Updated LXC Template List
 |unable to create CT 101 - command 'lxc-usernsexec -m u:0:100000:65536 -m g:0:100000:65536 -- tar xpf - --zstd --totals --one-file-system -p --sparse --numeric-owner --acls --xattrs '--xattrs-include=user.*' '--xattrs-include=security.capability' '--warning=no-file-ignored' '--warning=no-xattr-write' -C /var/lib/lxc/101/rootfs --skip-old-files --anchored --exclude './dev/*'' failed: exit code 2
 /bash: line 182: exit: A problem occured while trying to create container.: numeric argument required
 -ot@proxmox:~# `

**Comment by tteck** [comment]
My _guess_ is it's your storage.

**Reply by Calzor-Suzay** [reply]
I've deployed HAOS using same helper scripts fine in the past.
What can I look at/investigate?

In proxmox logs I seee
  Logical volume "vm-101-disk-0" created.
Creating filesystem with 524288 4k blocks and 131072 inodes
Filesystem UUID: 03e1f85f-15f9-4268-8c59-150de13f2395
Superblock backups stored on blocks: 
	32768, 98304, 163840, 229376, 294912
extracting archive '/mnt/pve/qNap/template/cache/debian-12-standard_12.2-1_amd64.tar.zst'
/*stdin*\ : Decoding error (36) : Data corruption detected 
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
  Logical volume "vm-101-disk-0" successfully removed.

TASK ERROR: unable to create CT 101 - command 'lxc-usernsexec -m u:0:100000:65536 -m g:0:100000:65536 -- tar xpf - --zstd --totals --one-file-system -p --sparse --numeric-owner --acls --xattrs '--xattrs-include=user.*' '--xattrs-include=security.capability' '--warning=no-file-ignored' '--warning=no-xattr-write' -C /var/lib/lxc/101/rootfs --skip-old-files --anchored --exclude './dev/*'' failed: exit code 2

and on my qNap CT templates I see "debian-12-standard_12.2-1_amd64.tar.zst"

**Reply by tteck** [reply]
Use local-lvm as a test.

**Reply by tteck** [reply]
> extracting archive '/mnt/pve/qNap/template/cache/debian-12-standard_12.2-1_amd64.tar.zst'
/stdin\ : Decoding error (36) : Data corruption detected

**Comment by Calzor-Suzay** [comment, marked as answer]
I got it working....

​I removed the template, then manually downloaded it via Proxmox off the list.

Ran the script again and it used the one I downloaded manually and installed fine.
